setup.py

```diff
@@ -43,13 +43,30 @@
 
     BOKEHJS_ACTION="install" pip install .
 '''
-from setuptools import find_packages, setup
+from setuptools import find_namespace_packages, setup
+
+import sys
+from pathlib import Path
+
+ROOT = Path(__file__).resolve().parent
+sys.path.append(str(ROOT))  # avoid depending on `build_meta:__legacy__`
 
 import versioneer
-from _setup_support import INSTALL_REQUIRES, BuildJSCmd, check_python
+from _setup_support import INSTALL_REQUIRES, BuildJSCmd, DevelopWithJs, check_python
+
+try:
+    from setuptools.command.build import build  # future-proof
```

Comment:
Right now there is no `setuptools.command.build`, but this is likely to change in the near future.

Reply:
I did notice that it was missing, I am glad to hear it will hopefully be coming. However, rather than this kind of clutter, we would simply use the working version now unconditionally, and if/when a better option comes along we will change `setup.py` to use that along with raising the min version of `setuptools` required.

Reply:
I guess probably the latter. Especially since some of my initial optimism has worn off. I think I might back off this attempt, and work incrementally from a different direction. Once the new `conda-build` comes out in month or two, I think we can stop using `load_setup_py_data` and move all metadata to `pyproject.toml`. Separating those concerns will let the `setup.py` work proceed more independently.